apps/microtvm/zephyr/template_project/src/mlperftiny/submitter_implemented.cc

```diff
@@ -19,6 +19,7 @@
 
 #include "api/submitter_implemented.h"
 
+#include <assert.h>
```

Comment:
is this needed?

Reply:
yeah, it is used in `TVMBackendAllocWorkspace`